### ISSUE TITLE
Reproducing results obtained by tuning

### ISSUE THREAD

**Issue by cboettig** [boettiger-lab/conservation-agents#5]
https://github.com/boettiger-lab/conservation-agents/blob/ddaa419280ea3564c8757dc0948448b8dfc165e4/tuning/utils/exp_manager.py#L611

Tuning logs the hyper-parameters tried and the resulting value obtained. However, using the recorded hyperparameters to reproduce the training run results is not trivial, even when the environment is not stochastic.

Tuning doesn't tune all possible hyper-parameters, but uses the sampling strategy outlined in https://github.com/boettiger-lab/conservation-agents/blob/master/tuning/utils/hyperparams_opt.py for each algo (note this version has already diverged from the corresponding one in the official Zoo repos).  

Mapping the logged parameters to the corresponding arguments is part of the challenge, another component is determining how parameters which are not tuned are set.  Some of these appear to have hard-wired values assigned by the tuning script (`ortho_init = False` in PPO, but is tuned in A2C).  Others are not mentioned in the tuning script at all, and are presumably either left at their defaults or use the value set in `hyperparams/<ALGO>.yml`  -- not sure which!  Guessing that `hyperparams` yamls files act as an over-ride for non-tuned defaults (?? you can name tunable params there too but they get tuned anyway), but not sure.  i.e. if you put `use_sde = True` in `hyperparams` yaml files, not clear to me if tuning occurs using that always, or using the algo default always (`use_sde = False`, the default for all algos), since it is never tuned by any of the sampling strategies.  

Note also that tuning does not set seed:
https://github.com/boettiger-lab/conservation-agents/blob/ddaa419280ea3564c8757dc0948448b8dfc165e4/tuning/utils/exp_manager.py#L611
